src/chemistry/chemTabModel.cpp

```diff
@@ -62,6 +59,22 @@ ablate::chemistry::ChemTabModel::ChemTabModel(std::filesystem::path path) {
     inputFileStream.open(ipath.c_str(), std::ios::in);
     LoadBasisVectors(inputFileStream, speciesNames.size(), iWmat);
     inputFileStream.close();
+    //load source energy scaler
+    sourceEnergyScaler = (PetscReal *) malloc(2*sizeof(PetscReal));
```

Comment:
I think that we missed freeing the sourceEnergyScaler

Reply:
Will free the variable.